buildSrc/src/main/java/Versions.kt

```diff
@@ -6,7 +6,7 @@ object Versions {
     const val ANDROID_TARGET_SDK_VERSION = 33
 
     const val ANDROID_VERSION_CODE = 1
-    const val ANDROID_VERSION_NAME = "0.1.0"
+    const val ANDROID_VERSION_NAME = "1.0.0"
```

Comment:
Wait, since we are development phase, I we should only increase the `minor` version -> e.g. `0.2.0`. Please prefer to [this](https://nimblehq.co/compass/development/version-control/release-management/) for more information about our team convention on [Semantic Versioning](https://semver.org/) 🙏
```suggestion
    const val ANDROID_VERSION_NAME = "0.2.0"
```

Reply:
Hmm, but our milestone was named `1.0.0` 🤔 That means do we need to change our milestone from `1.0.0` to `0.2.0` too @minhnimble 🤔

Reply:
yes, that is correct that we need to update the milestones too @thiennguyen0196, since we need more than 1 sprint to implement the initiative. When we use `1.0.0`, it is normally for making the first production release to the public 🙏 
cc @lydiasama